Backend/README.md

```diff
@@ -0,0 +1,82 @@
+# jasminegraph-ui
+
+Web based User Interface for JasmineGraph Distributed Graph Database Server
+
+## Overview
+
+jasminegraph-ui is a web-based user interface for interacting with the JasmineGraph Distributed Graph Database Server. This application is built using Next.js to provide a responsive and efficient frontend experience.
+
+## Prerequisites
+
+- Node.js (>= 12.x.x)
+- npm (>= 6.x.x) or yarn (>= 1.x.x)
+- Docker (optional, if you prefer to run using Docker)
+
+## Installation
+
+Clone the repository:
+
+```bash
+git clone https://github.com/jasminegraph/jasminegraph-ui.git
+cd jasminegraph-ui/Backend
+```
+
+Install dependencies:
+
+Using npm:
+
+```bash
+npm install
+```
+
+Using yarn:
+
+```bash
+yarn install
+```
+
+### Running the Application Locally
+
+To run the application locally, use the following command:
+
+Using npm:
+
+```bash
+npm run start
+```
+
+Using yarn:
+
+```bash
+yarn dev
+```
+
+### Building and Running with Docker
+
+To build and run the application using Docker, follow these steps:
+
+1. Ensure Docker is installed and running on your machine.
```

Comment:
Is there specific version/minimum version of Docker we expect to be there? We should be more specific here in terms of Docker version to use.

Reply:
Our application doesn’t depend on a specific Docker version, so any recent stable release should work fine.